GOVERNANCE.md

```diff
@@ -3,16 +3,31 @@
 This document describes the governance process under which the Serverless
 Working Group (WG) will manage this repository.
 
-## Working Group Meetings
+## Working Group Process
 
 In order to provide equitable rights to all Working Group members,
 the following process will be followed:
 
 * Official WG meetings will be announced at least a week in advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official WG meetings.
-* During meetings, priority will be given to PRs that appear to be ready for
-  a vote over those that appear to require discussions.
+* Proposed changes to any document will be done via a Pull Request (PR)
+* PRs will be reviewed in advance of WG meetings
+* Active members will vote on PRs in advance of Working Group meetings to
+  promote them to active consideration
+* Changes to the [spec.md](spec.md) require active member participation:
+  * if there are no negative votes, then any PR with 3 affirmative votes can be
+    raised for consideration
```

Comment:
While I suspect this line is there to force people to do something before the meetings, it feels more like it would have a bad negative effect because it means I can't schedule an PR for discussion that I'm pretty sure will be accepted if no one bothers to LGTM it.  I think I (or anyone else who wants to add a topic to the agenda) should have that freedom to understand the dynamics and sentiment of the group w.r.t. certain topics.

As an example, the logging level attribute PR had pretty much no comments yet I could guess that we'd come to a pretty quick agreement on what to do with it based on previous conversations. I think scheduling that one for our quick chat was the right thing to do - but this would have prevented me from doing that.

Reply:
The goal is to create momentum for interoperability.  The current undocumented process of scheduling unobjectionable PRs, tends to prioritize optional attributes, which does nothing to moving us toward a 0.1 specification

Reply:
Well, to be fair,I think that's only because most of our meaty attributes were blocked by several weeks of "what is source?". That didn't leave us with a lot of other things to discuss. Hopefully that will be behind us soon.

Reply:
I believe that "what is source?" discussion would have been accelerated by this process.  We could have focused discussion on proposals where WG members were aligned -- encouraging people to address issues from minority dissenters in the PR in order to turn a "no" vote into a "yes" and focus discussion on proposals that had alignment.  In that case, I think the dissent was all about the lack of clarity in definition and this process would have moved us forward quicker. 

If no PRs are ready, then we could use WG meeting for discussion of issues to create greater alignment and accelerate forward process, rather than using the time to discuss lower priority issues.